workers/social/lib/social/models/account.coffee

```diff
@@ -1358,33 +1361,40 @@ module.exports = class JAccount extends jraphical.Module
   ###
   fetchOtaToken: secure (client, callback) ->
 
-    { sessionToken } = client
+    { sessionToken, username } = client
 
     errorCallback = ->
       callback new KodingError 'Invalid session'
 
     return errorCallback()  unless sessionToken
 
-    @isEmailVerified (err, isVerified) ->
+    @isEmailVerified (err, isVerified) =>
 
       return callback err  if err
 
       if not isVerified
-        message = 'Sorry, you need to confirm your email address first.'
-        return callback new KodingError message
+        @findUserAndConfirmEmail username, (err) =>
```

Comment:
instead of verifying it in here automatically let's remove this check block from here completely. we'll update old data for existing users on production.